folly/Utility.h

```diff
@@ -384,4 +385,13 @@ constexpr auto to_unsigned(T const& t) -> typename std::make_unsigned<T>::type {
   return static_cast<U>(t);
 }
 
+template <class E>
+FOLLY_NODISCARD constexpr auto to_underlying_type(E e) noexcept
+{
```

Comment:
Note: I would auto-reformat this anyway after importing, but `{` would go on the previous line.

Reply:
Reformatted to put the brace on the previous line.